Source/WebCore/platform/graphics/skia/PlatformDisplaySkia.cpp

```diff
@@ -82,7 +92,111 @@ static sk_sp<const GrGLInterface> skiaGLInterface()
 
 static thread_local RefPtr<SkiaGLContext> s_skiaGLContext;
 
-unsigned initializeMSAASampleCount(GrDirectContext* grContext)
+#if USE(LIBDRM)
+static inline bool isNewIntelDevice()
```

Comment:
It feels awkward to me that such quirks are maintained in a random file in the Skia backend - can we move it at least to a standalone file elsewhere? SkiaQuirks.cpp?

Reply:
This is where we need this one, if we end up adding more quirks for other things then we could move it to a different file.